clientv3/client.go

```diff
@@ -86,6 +86,7 @@ func NewFromConfigFile(path string) (*Client, error) {
 // Close shuts down the client's etcd connections.
 func (c *Client) Close() error {
 	c.cancel()
+	c.Watcher.Close()
```

Comment:
Wouldn't `c.conn.Close()` close the watcher as well? Why didn't we have this before?